crates/re_viewport/src/viewport_blueprint.rs

```diff
@@ -46,8 +47,18 @@ pub struct ViewportBlueprint<'a> {
 }
 
 impl<'a> ViewportBlueprint<'a> {
+    pub fn is_invalid(&self) -> bool {
+        !self.space_views.is_empty()
+            && self
+                .space_views
+                .values()
+                .all(|sv| sv.class_name() == &SpaceViewClassName::invalid())
```

Comment:
it's only invalid if _all_ are invalid? What if 9/10 are invalid?

Reply:
My thought process was that if there are some views that are valid, then we have *something* to show, so I don't want to just throw it away.  The user can always manually click the reset button in that case, or just remove the invalid views to create new ones.

Reply:
Fair - let's add that as a comment. Are the invalid views culled somewhere?

Reply:
There's no culling yet, though we could add it. At the moment, instead the layout remains unchanged, but the invalid view just shows a warning message. My thought was that once we get to a point where only some views can be invalid, it's better to make the user aware that something went wrong so they can handle it than to just magically cull them behind the scenes.